packages/integration-ftp/src/index.ts

```diff
@@ -1,4 +1,5 @@
 import taskRnvVercelDeploy from './tasks/task.rnv.ftp.deploy';
+//@ts-ignore
```

Comment:
😁

Reply:
imports of root level json files (outside of src) not supported by TS in strict configuration. so I had to ts-ignore all of package.json files present in TS codebase. but we could change it to ts-expect-error potentially